analysis/create_tables.R

```diff
@@ -20,7 +20,9 @@ df_demographics_counts <- map_dfr(
     # summarises df with a new column which counts occurences (n)
     summarise(n = n()) %>%
     mutate(category = .x) %>%
-    rename(subcategory = 1)
+    rename(subcategory = 1) %>%
+    filter(n > 7) %>%
+    mutate(n = (round(n / 5) * 5))
 )
```

Comment:
I think we can delete some of these brackets?
```suggestion
    mutate(n = round(n / 5) * 5)
```